Mathlib/Data/Quot.lean

```diff
@@ -91,9 +91,10 @@ theorem lift_mk (f : α → γ) (h : ∀ a₁ a₂, r a₁ a₂ → f a₁ = f a
 #align quot.lift_beta Quot.lift_mk
 
 @[simp]
-theorem lift_on_mk (a : α) (f : α → γ) (h : ∀ a₁ a₂, r a₁ a₂ → f a₁ = f a₂) :
+theorem liftOn_mk (a : α) (f : α → γ) (h : ∀ a₁ a₂, r a₁ a₂ → f a₁ = f a₂) :
   Quot.liftOn (Quot.mk r a) f h = f a :=
   rfl
+#align lift_on_mk liftOn_mk
```

Comment:
I think `#align` does not add the namespace you are currently in to the alignment, but since I did not get `mathport` to run I am not completely sure.

Reply:
Yes, aligns need to be fully qualified on both the lean 3 and lean 4 sides.

Reply:
Fixed.